src/Axis.js

```diff
@@ -305,7 +306,10 @@ export default class Axis extends BaseClass {
         if (t !== "1") n = `${t} x ${n}`;
         return d < 0 ? `-${n}` : n;
       }
-      return this._d3Scale.tickFormat ? this._d3Scale.tickFormat(labels.length - 1)(d) : d;
+
+      let n = this._d3Scale.tickFormat ? this._d3Scale.tickFormat(labels.length - 1)(d) : d;
+      n = n.replace(/[^\d\.\-eE+]/g, "") * 1;
```

Comment:
Do we really need to catch for `/eE` here? Wouldn't that only appear in a log scale (and log scales get dealt with separately a few lines above).

Also, it appears the ` * 1;` is turning the variable into a Number, so we may not need the `parseFloat` in the line below.